Scripts/XRTrailRenderer.cs

```diff
@@ -304,104 +256,123 @@ public void EditorCheckForUpdate()
         Initialize();
     }
 
-    // TrailRenderer Functions
     /// <summary>
     /// Removes all points from the TrailRenderer. Useful for restarting a trail from a new position.
     /// </summary>
     public void Clear()
     {
+        var zeroVec = Vector3.zero;
+        var zeroColor = new Color(0, 0, 0, 0);
```

Comment:
you can just use `Color.clear` i think, should be the same: 
https://docs.unity3d.com/ScriptReference/Color-clear.html